src/drive/mobile/modules/settings/components/SelectMediaBuckets.jsx

```diff
@@ -0,0 +1,108 @@
+/* global cozy */
+import React from 'react'
+import { connect } from 'react-redux'
+import Main from 'drive/web/modules/layout/Main'
+import ScrollToTop from 'drive/web/modules/navigation/ScrollToTop'
+import styles from '../styles'
+import Toggle from 'cozy-ui/react/Toggle'
+import { getMediaBuckets, addMediaBucket, delMediaBucket } from '../duck'
+import { getMediaBuckets as getAllMediaBuckets } from 'drive/mobile/lib/media'
+
+const { BarCenter } = cozy.bar
+
+class SelectMediaBuckets extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      mediaBuckets: []
+    }
+  }
+
+  createHandle(item) {
+    const self = this
+    return checked => {
+      if (checked) {
+        self.props.addMediaBucket(item)
+      } else {
+        self.props.delMediaBucket(item)
+      }
+    }
+  }
+
+  handleChange(e) {
+    const item = e.target.id
+    const isChecked = e.target.checked
+    if (isChecked) {
+      this.setState(state => ({
+        selectedBuckets: state.selectedBuckets.add(item)
+      }))
+    } else {
+      this.setState(state => {
+        const newSelectedBuckets = new Set(state.selectedBuckets)
+        newSelectedBuckets.delete(item)
+        return {
+          selectedBuckets: newSelectedBuckets
+        }
+      })
+    }
+  }
+
+  componentDidMount() {
+    var self = this
+    getAllMediaBuckets().then(data => {
+      self.setState({
+        mediaBuckets: data
+      })
+    })
+  }
+
+  render() {
+    return (
+      <Main>
+        <ScrollToTop />
+        <BarCenter>
+          <h2 className={styles['settings__title']}>Select Media Buckets</h2>
```

Comment:
You started to use string here : https://github.com/cozy/cozy-drive/pull/1677/files#diff-496c2803b6373b077bd43538e90ef51cR362 

To be able to use it, you have to : 
```jsx 
Component {
 const { t } = this.props 
}
export translate()(Component)
```

The translate HOC will inject a `t` props. And after that you can use : 
t('MyString')

Reply:
Fixed